CONTRIBUTING.md

```diff
@@ -0,0 +1,183 @@
+
+#### Examples
+
+```
+feat: add send method to bot class
+
+This commit adds a new method to the bot class, which allows you to send messages to the chat.
+
+Closes #1
+```
+
+```
+
+refactor!: rename send method to sendMessage
+
+BREAKING CHANGE: send method has been renamed to sendMessage
+
+This commit renames the send method to sendMessage, to make it more clear what the method does.
+
+Closes #2
+```
+
+```
+
+fix(sendMessage): make sendMessage not accept undefined
+
+This commit fixes a bug where sendMessage would accept undefined as a message, and would send it to the chat.
+
+Closes #3
+```
+
+### Pull Request Template
+
+When you create a Pull Request, you will be presented with a template. This template is there to help you write a good Pull Request. It is also there to help me understand what you are trying to do. So I strongly request that you use that template and do not replace anything but the summary content and checklist content.
+
+### Code Style
+
+Evangeline uses [Prettier](https://prettier.io/) to format the code. This means that all code must be formatted using Prettier. If you are using VSCode, you can install the [Prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) to automatically format your code.
+
+### Linting
+
+Evangeline uses [ESLint](https://eslint.org/) to lint the code. This means that all code must be linted using ESLint. If you are using VSCode, you can install the [ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) to automatically lint your code.
+
+### Testing your change(s)
+
+Testing your change is quite straightforward. You can simply run
+
+```bash
+yarn link
+yarn link <path/to/evangeline>
+```
+
+This will link your local version of evangeline to your project. You can then test your changes. Once you are done, you can simply run
+
+```bash
+yarn unlink <path/to/evangeline>
+```
+
+This will unlink your local version of evangeline from your project.
+
+## Submitting an Issue
+
+### Issue Template
+
+When you create an issue, you will be presented with a template. This template is there to help you write a good issue. It is also there to help me understand what you are trying to do. So I strongly request that you use that template and do not replace anything in the template and only describe your issue.
+
+## Documentation
+
+Evangeline uses [TypeDoc](https://typedoc.org/) to generate documentation. This means that all documentation must be written using TypeDoc.
+
+To add a documentation, simply add a docstring above your code. For example:
+
+```ts
+/**
+ * This is a function that does something
+ * @param bar The bar to foo
+ * @returns The foobar
+ */
+function foo(bar: string) {
+  return bar;
+}
+```
+
+This _does_ make use of [JSDoc](https://jsdoc.app/). However, it is not the same as JSDoc. You can read more about the differences [here](https://typedoc.org/guides/doccomments/).
```

Comment:
JSDoc is not the same as JSDoc? you mean typedoc?

Reply:
https://github.com/eludris-community/evangeline/pull/9/commits/58bc0acfd2e41e500d49509109d2c163baf3e75a